extension/src/openvic-extension/singletons/GameSingleton.cpp

```diff
@@ -257,9 +249,10 @@ Error GameSingleton::_update_colour_image() {
 	Error err = OK;
 
 	InstanceManager const* instance_manager = get_instance_manager();
-	if (instance_manager != nullptr && !get_definition_manager().get_mapmode_manager().generate_mapmode_colours(
-		instance_manager->get_map_instance(), mapmode, colour_data_array.ptrw()
-	)) {
+	if (instance_manager != nullptr &&
+		!get_definition_manager().get_mapmode_manager().generate_mapmode_colours(
+			instance_manager->get_map_instance(), mapmode, colour_data_array.ptrw()
+		)) {
```

Comment:
The lack of a line break before the `) {` makes the indentation look weird, with the `if` statement's body being the same indentation level as the line above it